lib/src/base-styles/_nx-colors.scss

```diff
@@ -289,7 +289,15 @@
   --nx-color-system-notice-text: var(--nx-color-text);
 
   --nx-color-chart-line: var(--nx-swatch-grey-40);
-  --nx-color-chart-text: var(--nx-color-text-dark);
+  --nx-color-chart-text: var(--nx-color-text-stark);
+  --nx-color-chart-data-1: var(--nx-swatch-teal-40);
+  --nx-color-chart-data-2: var(--nx-swatch-red-65);
+  --nx-color-chart-data-3: var(--nx-swatch-green-35);
```

Comment:
In Kaleido, the the green is a `green-80`.

Reply:
> In Kaleido, the green is a `green-80`.

Oh nevermind, the color is a `green-35`, but the label says `green-80`. I will let Jenni know.

Reply:
> Oh nevermind, the color is a green-35, but the label says green-80. I will let Jenni know.

@cindyko226 already asked about that this morning in the channel